harmony/harmony.py

```diff
@@ -361,6 +362,57 @@ def _params(self, request: Request) -> dict:
 
         return params
 
+    def _headers(self) -> dict:
+        """
+        Create (if needed) and return a dictionary of headers.
+        Code partially adaped from:
+            https://github.com/requests/toolbelt/blob/master/requests_toolbelt/utils/user_agent.py
+        """
+        if 'headers' not in self.__dict__:
+            session = self._session()
+            existing_user_agent_header = session.headers.get('User-Agent')
+            if existing_user_agent_header:
+                user_agent_content = set([existing_user_agent_header])
+            else:
+                user_agent_content = set([])
+
+            # Get harmony package info
+            try:
+                import harmony
+                user_agent_content.add(f'harmony-py/{harmony.__version__}')
+            except Exception as e:
+                print("Following exception was caught "
+                      "when building user-agent headers for harmony-py:")
+                print(e)
+
+            # Get platform info
+            try:
+                p_system = platform.system()
```

Comment:
Just FYI - that with the harmony and harmony-service-lib PRs I don't think we're planning on adding OS type information to the user agent. We'll probably just have something like `harmony/0.0.0 harmony-uat`.

Reply:
Thanks again Chris. Actually James pointed me the comment in JIRA story 767 with platform info included (https://bugs.earthdata.nasa.gov/browse/HARMONY-767), but please let me if you want me remove it.

Reply:
I think it makes sense you are providing that info especially for a client library. Maybe we should do the same thing for the harmony application too with the other ticket.

Reply:
Did a cleaner import ("`from harmony import __version__ as harmony_version`"), but still a little bit reluctant to put on the top of file (worries breaking other things if we can't import).